mm2src/coins/utxo/utxo_withdraw.rs

```diff
@@ -25,6 +25,18 @@ use utxo_signer::sign_params::{OutputDestination, SendingOutputInfo, SpendingInp
 use utxo_signer::{with_key_pair, UtxoSignTxError};
 use utxo_signer::{SignPolicy, UtxoSignerOps};
 
+/// Num of blocks applied to withdrawal transactions
+/// that have a high priority, indicating a need for faster confirmation.
+pub const HIGH_TX_FEE: u8 = 1;
+
+/// Num of blocks applied to withdrawal transactions
+/// that have a normal priority, indicating a moderate confirmation time.
+pub const NORMAL_TX_FEE: u8 = 2;
+
+/// Num of blocks applied to withdrawal transactions
+/// that have a low priority, indicating a longer confirmation time.
+pub const LOW_TX_FEE: u8 = 4;
```

Comment:
I have some concerns about the choice of the number of blocks used for transaction confirmation priority. 

For high fee transactions, we could consider using 2 blocks. This is based on the below https://github.com/KomodoPlatform/komodo-defi-framework/blob/9ec826a1c0f74664cce4e0cb383b79455335aec1/mm2src/coins/utxo/rpc_clients.rs#L2012-L2016 However, this might only be applicable to some UTXO coins and not all. For Bitcoin, the situation might be different. Therefore, it might be beneficial to get these values from the coin config where they are tested separately for each coin that supports fee estimation.

For normal fee transactions, 6 blocks seem to be a good choice. This implies that we expect the transaction to be confirmed within an hour for Bitcoin.

For low fee transactions, the goal should be to choose the lowest possible fee that will still get the transaction confirmed and not removed from the mempool. This is easier said than done. The [mempool explorer](https://mempool.space/docs/api/rest#get-recommended-fees) provides a good API for estimating this for BTC only. It's the `economyFee` field, which is the lower of the low priority (1 hour fee) or 2x the minimum allowed fee. 

I recommend testing the value generated from Electrum's `blockchain.estimatefee` against the values from the mempool explorer. The mempool explorer is one of the most reliable explorers for BTC fees. If we can't estimate good fees using Electrum for BTC, we can use the mempool API for BTC only. Although it's rate-limited, it will be rate-limited per user which is acceptable.

Lastly, please also modify the below doc comments to reflect the number of blocks used. https://github.com/KomodoPlatform/komodo-defi-framework/blob/9ec826a1c0f74664cce4e0cb383b79455335aec1/mm2src/coins/lp_coins.rs#L1916-L1925

P.S. We can continue this discussion on DM since it might be a bit complicated.

Reply:
This is how mempoll explorer calculates the different fee levels https://github.com/mempool/mempool/blob/db34ca6a5f37f8c420ce961f0ccd96bba9205dfd/backend/src/api/fee-api.ts#L22-L63 I guess High = 1 , Normal = 2, Low = 3 or 4 is applicable to bitcoin only and not other UTXOs.

Reply:
Only last note :)

In the future we can enable RBF and use lower fees for the low level while giving the ability to the user to bump the fee if it's not confirming in time. This is out of scope of this PR ofcourse.

Reply:
done